types/barnard59-base/barnard59-base-tests.ts

```diff
@@ -56,6 +62,7 @@ function testFlatten() {
 }
 
 function testForEach() {
+    // prettier-ignore
```

Comment:
Why are the `prettier-ignore` necessary? Looks like there will be many across the types packages